provider/datagen/src/transform/segmenter/mod.rs

```diff
@@ -773,9 +773,14 @@ impl crate::DatagenProvider {
 
```

Comment:
Nit: Can you assert somewhere that the datagen builtin dictionaries stay consistent with the icuexport dictionaries and don't drift out of sync?

Reply:
The download script replaces them, and it's diff-checked in CI